source/manual/get-started.html.md

```diff
@@ -29,12 +54,179 @@ We use a Docker environment - [govuk-docker][] - for local development.
+
+While the applications are available directly via the public internet, SSH access to remote environments is via a ‘jumpbox’. You’ll need to configure your machine to use this jumpbox and use `govukcli` to SSH into server.
+
+Copy the [example SSH config file][ssh-config] into the `~/.ssh/config` file on your host machine.
+
+Test that it works by running:
+
+    mac$ govukcli set-context integration
+    mac$ govukcli ssh backend
+
+You should be able to do the same thing in your VM:
+
+    dev$ govukcli set-context integration
+    dev$ govukcli ssh backend
+
+The built-in key-forwarding should mean that you don't need to edit the `~/.ssh/config` file inside the VM (it will default to your host machine's config file instead).
+
+#### Running a console
+Once you have SSH'd into a machine, you can also open a console for a particular application so you can execute commands, for example:
+
+```
+govuk_app_console publishing-api
+```
+
+[ssh-config]: https://github.com/alphagov/govuk-puppet/blob/master/development-vm/ssh_config
 
 ## 4. Get AWS access
 
-👉 [Setup your AWS account](/manual/set-up-aws-account.html)
+To work with [govuk-aws](https://github.com/alphagov/govuk-aws) and [govuk-aws-data](https://github.com/alphagov/govuk-aws-data),
+you will require an account in AWS.
+
+### Request a GDS AWS account
+
+GDS maintains a central account for AWS access. You will need to request an account from the Technology and Operations team.
+
+👉 [Request an AWS account](https://gds-request-an-aws-account.cloudapps.digital)
+
+You'll want to click "Request user access" - NOT "Request an account". After submitting the form, you should receive an email to say your account creation is in progress, and later another email saying the work has been completed. You can then move onto step 2.
+
+### Sign in to AWS
+
+To sign in, go to [the GDS AWS Sign page](https://gds-users.signin.aws.amazon.com/console), and use the following credentials:
+
+- "Account ID or alias": `gds-users`
+- Username: your `@digital.cabinet-office.gov.uk` email address
+- Password: your password
+
+👉 [Sign in to AWS GDS account](https://gds-users.signin.aws.amazon.com/console)
+
+### Set up your MFA
+
+You have to set up [Multi-Factor Authentication (MFA)][MFA].
+
+1. [Sign in to AWS GDS account](https://gds-users.signin.aws.amazon.com/console)
+2. Select or go to IAM service.
+3. Click on "Users" in the menu bar on the left hand side
+4. Enter your name
+5. Click on the link for your email address
+6. Click on the security credentials tab
+7. Click on the "Manage" link next to "Assigned MFA device"
+8. Follow the steps to set up your MFA device
+
+### Generate a pair of access keys
+
+You have to generate an AWS Access Key and Secret Key to be able to
+perform operations with AWS on the command-line.
+
+1. [Sign in to the gds-users AWS Console](https://gds-users.signin.aws.amazon.com/console).
```

Comment:
Can move this URL into an alias and call it from lines 164 and 178, rather than duplicate the URL - just makes it clearer to the doc maintainer that they refer to the same place.

Reply:
Sure!

Reply:
Done!